ang/civicase/case/list/directives/case-list-table.directive.html

```diff
@@ -12,8 +12,8 @@
   <div
     class="panel panel-default civicase__case-list-panel"
     ng-class="{'civicase__case-list-panel--summary': viewingCase, 'civicase__case-list-panel--focused': caseIsFocused}">
-    <div class="civicase__case-list-table-container" civicase-sticky-table-header>
-      <table ng-if="!caseIsFocused" class="table civicase__case-list-table" >
+    <div class="civicase__case-list-table-container">
+      <table ng-if="!caseIsFocused" class="table civicase__case-list-table" civicase-sticky-table-header>
```

Comment:
When was this error about `top` on the console was introduced? Did it happen in one of the previous backstop prs? because I never saw this error.

Reply:
Hey. After investigating. I found out that this error was introduced on https://github.com/compucorp/uk.co.compucorp.civicase/pull/164 (PR to branch `CATL-1250`) here https://github.com/compucorp/uk.co.compucorp.civicase/pull/164/commits/b1d847e716b524b3550a1fd462bebc2b4b516dea#diff-09fe1bcdaae295b1fa2d0bd83b04bacdR16

Since this is not on master. we are making the changes back to what it is on master.
Please check the PR details for more info.